tests/unit/models/project-test.js

```diff
@@ -29,7 +52,9 @@ describe('models/project.js', function() {
             baseURL: '/foo/bar'
           });
 
-          project = new Project(projectPath, { }, new MockUI());
+          project = new Project(projectPath, { }, new MockUI(), cli);
```

Comment:
Seems slightly odd to swap the pattern around from earlier tests, I was expecting to see:

```js
let cli = new MockCLI();
project = new Project(projectPath, { }, cli.ui, cli);
```